src/components/ClusterManager.js

```diff
@@ -527,6 +513,10 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
     const totalCost = _.sum(_.map(({ machineConfig, status }) => {
       return (status === 'Stopped' ? machineStorageCost : machineConfigCost)(machineConfig)
     }, clusters))
+    const disabled = !canCompute
```

Comment:
I realize this isn't your code, but as long as it's being moved, boolean variables should be named `isFoo` rather than `foo`

Reply:
I'll make the change, thanks.

Reply:
I'd just go for the explicit one.